recipes/charliecloud/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = "0.25" %}
+
+package:
+  name: charliecloud
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpc/charliecloud/releases/download/v{{ version }}/charliecloud-{{ version }}.tar.gz
+  sha256: 62d6fd211e3a573f54578e1b01d5c298f9788b7eaf2db46ac94c2dcef604cc94
+  patches:
+    # this hard-turns the flag OFF for building the tests (otherwise fail)
+    # it also hard-tuns the flag ON for ch-image.
+    - hardcode_build.diff
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - ./configure --prefix=$PREFIX
+    - make
+    - make install
```

Comment:
Is there a make check we can run here?

Reply:
Not really, but I will add it anyway. I will be doing more proper testing once this is a feedstock, but for that `--privileged` flag will be needed. 

```
ubuntu@primary:~/charliecloud-0.25$ make check
Making check in lib
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/lib'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/lib'
Making check in bin
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/bin'
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/bin'
Making check in doc
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/doc'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/doc'
Making check in examples
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/examples'
ln -fTs dirCb copy/dirCa/symlink-to-dirCb
ln -fTs fileDa copy/dirD/symlink-to-fileDa
ln -fTs dirEb copy/dirEa/symlink-to-dirEb
ln -fTs filey copy/dirG/s_dir2
ln -fTs diry copy/dirG/s_dir3
ln -fTs filey copy/dirG/s_file2
ln -fTs diry copy/dirG/s_file3
ln -fTs fileA copy/symlink-to-fileA
ln -fTs fileB copy/symlink-to-fileB-A
ln -fTs fileB copy/symlink-to-fileB-B
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/examples'
Making check in misc
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/misc'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/misc'
Making check in packaging
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/packaging'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/packaging'
Making check in test
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25/test'
ln -fTs /tmp fixtures/symlink-to-tmp
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25/test'
make[1]: Entering directory '/home/ubuntu/charliecloud-0.25'
make[1]: Nothing to be done for 'check-am'.
make[1]: Leaving directory '/home/ubuntu/charliecloud-0.25'
```